android/java/org/chromium/chrome/browser/sync/BraveAndroidSyncSettings.java

```diff
@@ -22,7 +22,7 @@ public BraveAndroidSyncSettings(Account account) {
     // next SyncController.updateSyncStateFromAndroid sync gets disabled, so
     // we act here as mChromeSyncEnabled would be true.
     // Following up issue https://github.com/brave/brave-browser/issues/10454
-    @Override
+    // @Override
```

Comment:
why is this commented instead of removing?

Reply:
It was temporary commit, it was removed later